### PR TITLE
Jason tests

### PR DESCRIPTION
**Pull request by jmmcnj**
pushing tests to increase coverage

### CODE REVIEW

File: DOL.WHD.Section14c.Web/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "karma start --single-run",
```
Comment (MrMatt57) [inline DOL.WHD.Section14c.Web/package.json]: need to leave --single-run on so the tests finish on the build server.

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (MrMatt57) [inline .vscode/launch.json]: Does this file work?  If not we should exclude it.

Comment (jmmcnj) [inline .vscode/launch.json]: Not sure, I dont think I was able to get it working actually, I should just remove it

Comment (MrMatt57) [inline .vscode/launch.json]: yea, let's take it out unless it works.

Comment (jmmcnj) [inline .vscode/launch.json]: do i need to revert the whole PR?

**Pull request merged**